poetry.lock

```diff
@@ -276,6 +296,43 @@ category = "dev"
 optional = false
 python-versions = "*"
 
+[[package]]
+name = "dsolve"
+version = "0.0.5"
+description = "Solver of dynamic equations with forward looking variables"
+category = "main"
+optional = false
+python-versions = "*"
+
+[package.dependencies]
+scipy = ">=1.9.0"
+
+[package.extras]
+dev = ["pytest (>=7.1.2)"]
+
+[[package]]
+name = "dwave-preprocessing"
+version = "0.3.2"
+description = "An API for preprocessing tools"
+category = "main"
+optional = false
+python-versions = ">=3.6"
+
+[package.dependencies]
+dimod = ">=0.10.0,<0.11.0"
```

Comment:
In this release, we won't yet include any pipeline to benchmark against Pyomo / QBSolv yet. Thus, we may leave out the requirement to install...
... dimod
... dwave-qbsolv
...pyomo